src/main.js

```diff
@@ -23,6 +23,8 @@ const settingsConfig = {
     ],
 };
 
+const Achievements = Home
```

Comment:
This line looks a little odd...?

Reply:
This is to give the button something to return to. Currently there's no achievements screen to transition to so the game will break before the transition and we get no (fake or otherwise) stat call. Not sure of a nicer way to do this though